kodev

```diff
@@ -981,6 +985,69 @@ OPTIONS:
     } && popd || exit
 }
 
+function kodev-check() {
```

Comment:
I do wonder if inlining it all like this is the best plan for ease of use/clarity/maintenance? I'm inclined to think it might be preferable to stick to something along the general lines of how it is right now. 

You could just put a make static-check here that executes the script, or make or not you could also just `source` it or something.
https://github.com/koreader/koreader/blob/ca6d82e668f20d0be167bf44f94a8b8dd8abf77c/Makefile#L606-L612